cmd_image_build.go

```diff
@@ -55,12 +79,50 @@ func init() {
 				return err
 			}
 
-			if argEntrypoint != "" {
+			if !reflect.ValueOf(flags.config).IsZero() {
```

Comment:
OK, I'll bite. Why exactly does this require `reflect`?

Reply:
Because you can't just `argEntrypoint == typeName{}` because it contains slices and you can't compare slices for equality because Go is dumb.

Reply:
Augh. OK.